### DISCUSSION TITLE
all directory paths prefixed with /

### DISCUSSION THREAD

**Discussion by danie-dejager** [dundee/gdu#245] [Polls]
I'm wondering if the` gdu -n` output for directories should have a / prefixed or not. Is it not enough that files and directories are a different color? What if the output instead uses 3 columns? First for directory (d) or file (f), then size and finally the name without a prefixed / for directories. I sometimes copy and paste the output from the CLI and needs to make sure I don't accidently select the /.

**Comment by dundee** [comment]
Interesting idea! It would be a breaking change, so we would need to introduce it probably as a new mode.

**Comment by rwahyudi** [comment]
Maybe use '/' if showing folder in root directory and ./ if we are in subfolders?

I think displaying the folder name for each item is unnecessarily repetitive and can easily clutter your screen. 
What I love to have is an option to display an additional line in the status footer to display the properties of the highlighted item.

The properties could include  permissions, ownership & timestamp similar to ls -lh 
`-rw-r--r-- 1 root root 158 Jun  8  2013 /etc/hosts`
